Packs/CommonPlaybooks/Playbooks/playbook-Email_Address_Enrichment_-_Generic_v2.1.yml

```diff
@@ -714,22 +718,18 @@ inputs:
 - key: Domain
   value:
     complex:
-      root: inputs.Domain
-      filters:
-      - - operator: isNotEmpty
-          left:
-            value:
-              simple: inputs.Domain
-            iscontext: true
+      root: Account.Email
```

Comment:
usually, that won't be a domain ... maybe regex can help here

Reply:
removed it